x-pack/plugins/ui_actions_enhanced/public/drilldowns/url_drilldown/url_drilldown_scope.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UrlDrilldownGlobalScope, UrlDrilldownScope } from './types';
+import { getFlattenedObject } from '../../../../../../src/core/public';
+
+export function buildScope<
```

Comment:
über nit: 

```suggestion
export function buildScopes<
```

Reply:
Explained may take on it here: https://github.com/elastic/kibana/pull/75450#discussion_r477219188
I think it should be `singular`